salt/states/chocolatey.py

```diff
@@ -460,3 +452,81 @@ def upgraded(
     ret["changes"] = salt.utils.data.compare_dicts(pre_install, post_install)
 
     return ret
+
+
+def source_added(name, source_location, username=None, password=None, force=False):
```

Comment:
To follow the Salt naming convention, please change this to `source_present`

Reply:
Thank you for your feedback. It should be better now.